library_generation/generate_pr_description.py

```diff
@@ -97,10 +95,13 @@ def get_commit_messages(
     selecting commit message. This commit should be an ancestor of
     :param paths: a mapping from file paths to library_name.
     :param is_monorepo: whether to generate commit messages in a monorepo.
+    :param repo_level_message: commit messages regarding repo-level changes.
     :return: commit messages.
     :raise ValueError: if current_commit is older than or equal to
     baseline_commit.
     """
+    if current_commit_sha == baseline_commit_sha:
```

Comment:
What is the behavior if current_commit_sha < baseline_commit_sha? It is not a valid scenario now, but maybe we need to revert the changes in some cases. The scripts should at least not blow up at this point.

Reply:
We can't compare different commit sha directly. We get the commit time of two commits and if the current commit is older than baseline commit, the script will exit with error (we have unit test to verify this behavior).

Reply:
> the script will exit with error

I remember it now. Thanks!